src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestIT.java

```diff
@@ -65,27 +66,16 @@ public void beforeAll() throws Exception {
     jdbcConnection
         .createStatement()
         .execute(String.format("create or replace table %s (c1 char(10));", TEST_TABLE));
-    jdbcConnection
-        .createStatement()
-        .execute("alter session set ENABLE_PR_37692_MULTI_FORMAT_SCANSET=true;");
-    jdbcConnection
-        .createStatement()
-        .execute(
-            String.format(
-                "alter database %s set ENABLE_PR_37692_MULTI_FORMAT_SCANSET=true;", TEST_DB));
-    jdbcConnection
-        .createStatement()
-        .execute(
-            String.format(
-                "alter table %s set ENABLE_PR_37692_MULTI_FORMAT_SCANSET=true;", TEST_TABLE));
-    jdbcConnection.createStatement().execute("alter session set ENABLE_UNIFIED_TABLE_SCAN=true;");
```

Comment:
I assume the reason for removing these are because the default are already set to the correct value?

Reply:
The default is set correctly and we can't actually set these parameters from the github test account